code-for-canada/project-thundercat
# Pull out css2

# Description

Pulled Tab and TabNavigation styles out of App.css. Removed some unused css; Replacing PR #95

## Type of change

Please delete options that are not relevant.

- Code cleanliness or refactor

## Screenshot

Looks the same as PR #94 

# Testing

Manual steps to reproduce this functionality:

1.  Go to Start Emib
2. Navigate through to TestTabs
3. Verify that pages look the same as they did previously

Screenshot of unit tests run passing:
![image](https://user-images.githubusercontent.com/2746350/53515786-d422c800-3a98-11e9-91ad-2206eff5a5cd.png)

# Checklist

Applicable for all code changes.
